tests/components/stream/common.py

```diff
@@ -15,13 +15,17 @@ def generate_h264_video():
     import numpy as np
     import av
 
+    path = os.path.dirname(os.path.abspath(__file__))
+    file_out = '{}/video.ts'.format(path)
+
+    if os.path.isfile(file_out):
```

Comment:
When would this not be the case? Isn't the file always there?

Reply:
Yes. This is effectively dead code. It is, however, the code used to generate the included video, so remains currently to help with troubleshooting.

Reply:
We should at least add a comment.